frontend/src/api/silly.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const getReactionCompartment = async ({ id, model, version }) => {
+  const params = { model, version };
+  const { data } = await axios.get(`/silly/${id}`, { params });
+  console.log('Enter data!');
+  return data;
+};
```

Comment:
Is this piece of code is only for testing?

Reply:
Haha yes, I did not intend to push that. Will remove it :+1: